README.md

```diff
@@ -356,3 +370,23 @@ class SomeClass {
     static Node *node_b = Node::_new(); // crashes
 };
 ```
+
+## Updating Headers
+
+See [**Versioning**](#versioning) for details on the Godot versions tracked by
+each branch of this repository.
+
+If the relevant branch is not up-to-date for your needs, or if you want to sync
+the headers with your own modified version of Godot, here is the update
+procedure used to sync this repository with upstream releases:
+
+- Compile [Godot Engine](https://github.com/godotengine/godot) at the specific
+  version/commit which you are using.
+- Use the compiled executable to generate the `api.json` file with:
+  `godot --gdnative-generate-json-api api.json`
+- Copy the file `modules/gdnative/gdnative_api.json` into `godot-headers`.
+- Copy the files and folders from `modules/gdnative/include` into `godot-headers`,
+  overwriting existing content. (To be sure to be in sync, you can delete the
+  folders of this repository first, then copy the upstream folders in place.)
+  Make sure that you compiled the correct Godot version so that the generated
+  `gdnative_api_struct.gen.h` is up-to-date.
```

Comment:
This doesn't really apply here, `godot-headers` is a submodule so updating is just a matter of syncing the submodule with the relevant https://github.com/godotengine/godot-headers commit or tag.

Reply:
Not if you're dealing with a custom build of Godot, you'd still need to replace those files in godot-cpp.